students/barecki_andrzej/lesson_01_basics/lesson_01_basics_all_cases.py

```diff
@@ -0,0 +1,122 @@
+# Lesson 1 (Input, print and numbers) and # Lesson 2 (Integer and float numbers)3
+import os
+import math
+
+print('Problem 1: Sum of three numbers')
+print('Set first number:')
+a = int(input())
+print('Set second number:')
+b = int(input())
+print('Set third number:')
+c = int(input())
+result = a + b + c
+print(str(result) + os.linesep)
+
+print('Problem 2: Area of right-angled triangle')
+print('Set length of the base of a right-angled triangle')
+b = int(input())
+print('Set length of the height of a right-angled triangle')
+h = int(input())
+print('The area of right-angled triangle is equal: ' + str((1 / 2) * b * h) + os.linesep)
+
+print('Problem 3: Hello, Harry!')
+print('Set user name: ')
+name = input()
+print('Hello, ' + str(name) + '!' + os.linesep)
+
+print('Problem 4: Apple sharing')
+print('Set number of students:')
+N = int(input())
+print('Set number of apples:')
+K = int(input())
+print('Each student get ' + str(K // N) + ' apples')
+print('In basket remains ' + str(K % N) + ' apples' + os.linesep)
+
+print('Problem 5: Previous and next')
+print('Set an integer:')
+a = int(input())
+print('The next number for the number ' + str(a) + ' is ' + str(a + 1) + '.')
+print('The previous number for the number ' + str(a) + ' is ' + str(a - 1) + '.' + os.linesep)
+
+print('Problem 6: School desks')
+
+
+def compute_desks(nr):
+    return (nr + (nr % 2)) >> 1
+
+
+a = 0
+i = 0
+while True:
```

Comment:
Ciężko się w pierwszej chwili domyślić, że jest to pętla, która ma się wykonać 3 razy. Przejrzyj ten wątek na SO:
https://stackoverflow.com/questions/2970780/pythonic-way-to-do-something-n-times-without-an-index-variable

W przypadku tak krótkiej pętli `for _ in range(3)` będzie jak najbardziej ok. Na marginesie: w pythonie 2 zamiast range należy użyć xrange.

Popraw proszę na rozwiązanie z forem.

Reply:
Poprawilem